server/src/test/java/org/elasticsearch/script/JodaCompatibleZonedDateTimeTests.java

```diff
@@ -232,8 +232,8 @@ public void testYearOfEra() {
     }
 
     public void testToString1() {
-        assertMethodDeprecation(() -> assertThat(javaTime.toString("YYYY/MM/dd HH:mm:ss.SSS"),
-            equalTo(jodaTime.toString("YYYY/MM/dd HH:mm:ss.SSS"))), "toString(String)", "a DateTimeFormatter");
+        assertMethodDeprecation(() -> assertThat(javaTime.toString("yyyy/MM/dd HH:mm:ss.SSS"),
```

Comment:
I am confused by reading this. Shouldnt one be in capital letters and one not in order for this to work?

Reply:
Java time cares significantly about `Y` vs `y`. Afaict, Joda time only uses `Y` vs `y` to exclude `0` and **negative numbers** (actually, the documentation I can find differs about whether or not `0` is a valid value, but...).

That said, `y` in joda is the "year" w/o the constraint, which is a closer map to Java `y`.